next.config.js

```diff
@@ -6,7 +6,27 @@ const env = {
 
 console.log(env);
 
-module.exports = {
+// module.exports = {
+//   env: env,
+//   assetPrefix: isProd ? '/web' : '',
+// }
+
+const withSass = require('@zeit/next-sass');
+const withCSS = require("@zeit/next-css");
+module.exports = withCSS(withSass({
   env: env,
-  assetPrefix: isProd ? '/web' : ''
-}
+  assetPrefix: isProd ? '/web' : '',
+   webpack (config, options) {
+       config.module.rules.push({
+           test: /\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$/,
+           use: {
+               loader: 'url-loader',
+               options: {
+                   limit: 100000
+               }
+           }
+       });
+
+       return config;
+   }
```

Comment:
The indentation here doesn't seem to match the rest of the project? (4 vs 2 spaces?)